docs/compare_collections.md

```diff
@@ -29,7 +29,7 @@ Two collections meet the criteria for order-relaxed identity if:
 
 1. the value of the `elements.total.a` and `elements.total.b` match, (the collections have the same number of elements).
 2. this value is the same as `elements.a_and_b.<attribute>` for all attributes (the content is the same)
-3. any entries in `elements.a_and_b-same-order.<attribute>` may be true (indicating the order matches) or false (indicating the order differs)
+3. entries in `elements.a_and_b-same-order.<attribute>` may be either all true (indicating the order matches) or all false (indicating the order differs)
```

Comment:
I think this is not correct. Here's an example: https://refget.databio.org/comparison/fLf5M0BOIPIqcfbE6R8oYwxsy-PnoV32/JPd9Y-hwnhGD7HPe3yka4Qtx2YsIL8tW

These have order-relaxed identity. the compare function says:

```
    "a_and_b_same_order": {
      "lengths": false,
      "names": false,
      "sequences": false,
      "sorted_name_length_pairs": true
    }
```

So, it is not true that the entries have to be either all true or all false. The original statement that any entries may be true or false is more lenient. Should it say, any **collated attributes** should be either all true or all false? I think uncollated attributes may be either true or false, independently.

Reply:
Yes, you are right. My point was that if, say, the order of `lenghts` are the same btw a and b, but not `names`, then the data is corrupt and should not fall inside the definition of an `order-relaxed` identity. I think restricting my statement to **collated attributes** as you suggest fixes the issue.

Reply:
When I suggested this, I believe I hadn't read the caveat at the end yet (Limitations...). With that limitation in mind, i.e. assuming that the "collatedness" of the data is intact, I think the original text is correct.